src/test/lib/ApiTest.cpp

```diff
@@ -1215,7 +1215,7 @@ struct CloseFromCallbackContext {
 void
 QuicTestConnectionCloseFromCallback() {
     for (uint16_t i = 0; i < 20; i++) {
-        CxPlatWatchdog Watchdog(2000);
+        CxPlatWatchdog Watchdog(20000);
```

Comment:
This shouldn't be necessary IMO.

Reply:
It shouldn't, I agree, but for some reason I've not gotten to the root cause of the additional delay, so I left this in here for the time being.